src/styles.scss

```diff
@@ -1,9 +1,18 @@
 /* You can add global styles to this file, and also import other style files */
+@import '~@angular/material/theming';
+
+@import "../node_modules/@angular/material/prebuilt-themes/deeppurple-amber.css";
+
+// always include only once per project
```

Comment:
What is meant by project since there are two of these comment in this project?

Reply:
/Albin :)

Reply:
The first file with the @include isn't used at the moment :)